internal/cli/cmd/apply/apply.go

```diff
@@ -35,11 +35,15 @@ func NewCommand(cfg config.CLIConfig, opt *option.Option) *cobra.Command {
 	}
 
 	cmd := &cobra.Command{
-		Use:   "apply [--project=project] -f (FILENAME)",
+		Use:   "apply -f FILENAME",
```

Comment:
No opinion here -- more of a question. Would it generally be more clear to use long flags in the usage strings?

Reply:
Think it depends on the flag type, for `--project` I think it's a good choice. However, for files `-f` is so common that I think people can figure this out based on the context itself.

Reply:
I can live with that.